src/components/create-user.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div class="login-wrapper">
```

Comment:
CPOP (copy paste oriented programming) careful bro! this is not a login like my component!

Reply:
fixed my bro!